### ISSUE TITLE
npm publishing

### ISSUE THREAD

**Issue by fpereira1** [bergie/passport-saml#219]
Hi,

There has been some changes merged to master, but a new version has not yet been released on npm.

Changes since release of `v0.15.0`: https://github.com/bergie/passport-saml/compare/525748647c8a8c6e79073bf8965c639c36f262b5~...master

The latest release `v0.15.0` on npm was in December last year. https://libraries.io/npm/passport-saml/versions

Do you have plans to release a new version to npm?

Thanks!

**Comment by feenst** [comment]
Bump. Any plans to publish the latest to npm?

**Comment by markstos** [comment]
@pdspicer Any plans for a 0.16 release soon? Except for a JShint violation in the last commit, there's consider interest in getting the next release out. 

The last Jshint violation was:

```
lib/passport-saml/saml.js: line 926, col 6, Missing semicolon.
```

**Comment by markstos** [comment]
I released 0.16.0 today (but the project could still use maintenance help-- I'm short on time as well).
